e2e/src/main/java/gov/hhs/cdc/trustedintermediary/e2e/HapiParserContext.java

```diff
@@ -0,0 +1,30 @@
+package gov.hhs.cdc.trustedintermediary.e2e;
+
+import ca.uhn.fhir.fhirpath.IFhirPathEvaluationContext;
+import jakarta.annotation.Nonnull;
+import jakarta.annotation.Nullable;
+import org.hl7.fhir.instance.model.api.IBase;
+import org.hl7.fhir.instance.model.api.IIdType;
+import org.hl7.fhir.r4.model.Reference;
+
+public class HapiParserContext implements IFhirPathEvaluationContext {
+
+    /**
+     * When a FHIR path includes the "resolve()" method, this function is called to parse that into
```

Comment:
Javadocs typically go above the class declaration

Reply:
Added it